### ISSUE TITLE
GPS Driver: supported GPS driver

### ISSUE THREAD

**Issue by peterwillcn** [pendulum-project/ntpd-rs#1484]
Is it supported here（ https://www.ntp.org/documentation/drivers/ ）What about the GPS driver? Especially the Motorola Oncore GPS Receiver, a GPS time receiver with very high accuracy.

**Comment by JamesB192** [comment]
I am not affiliated with the project, but this has been asked before. So synopsizing...

There are no plans of adding ntp refclocks to ntpd-rs. Instead, the intent is to add a chrony type unix socket driver (not SHM); the one could run the mlichvar/ntp-refclock project, which supports many of the reflclocks from ntpd.

**Comment by davidv1992** [comment]
Yes, we do intend to add a chrony type unix socket driver. Initial experiments on implementing this are currently ongoing and looking positive.
